### ISSUE TITLE
Group shares support

### ISSUE THREAD

**Issue by PVince81** [owncloud/ocis-reva#194]
- [ ] find/enable matching API tests to see where it's missing

@individual-it I guess this is likely hidden by https://github.com/owncloud/ocis-reva/issues/34 ?

**Comment by individual-it** [comment]
@PVince81 yes, to test/try sharing the test infrastructure needs to have some info about the groups
